app/models/dictionary_sign_asset.rb

```diff
@@ -0,0 +1,57 @@
+class DictionarySignAsset
+  attr_reader :asset_url
+
+  class S3Adapter
+    cattr_accessor :region, :access_key_id, :secret_access_key
+    self.region = ENV.fetch("DICTIONARY_AWS_REGION", ENV.fetch("AWS_REGION", nil))
```

Comment:
These match the client values configured in dictionary.rake, so if we _did_ extract some of this code into a gem, I'd probably advocate for creating some sort of `DictionarySign.client` method to consistently use these.

Reply:
Or in fact to have a `DictionarySign.download` type method